### PR TITLE
Remove noUiSlider and use pi-ui one

### PR DESCRIPTION
**Pull request by guilhermemntt**
This PR is a PoC of https://github.com/decred/pi-ui/pull/308. It will solve https://github.com/decred/decrediton/issues/2874 when approved. For while, ``pi-ui`` dependence in ``package.json`` has been changed to my forked repository, but ``package.json`` changes will be discarded after the merge in the official ``pi-ui`` repo.

### CODE REVIEW

**Comment by tiagoalvesdulce** [comment]
You may know it already but you can [yarn link](https://classic.yarnpkg.com/en/docs/cli/link/) local packages to develop locally, run `yarn start` on pi-ui and you can see the changes made there on decrediton with hot reloading. One thing is that the react version of both repos have to be exactly the same. So, usually do the following: go to `/decrediton/node_modules/react`, link it, go to `/pi-ui` and `yarn link react`, now link pi-ui and go to `/decrediton` and yarn link pi-ui.

**Comment by guilhermemntt** [comment]
Thanks for your suggestion @tiagoalvesdulce! Luckily, just small changes have been required to fit ``pi-ui`` slider into Decrediton. By now, this PR is ready for review. Thank you all. I'll wait for the reviews.

File: package.json [hunk 1]
```diff
@@ -286,8 +286,7 @@
     "mv": "^2.1.1",
     "node-abi": "^2.15.0",
     "node-addon-loader": "decred/node-addon-loader#master",
-    "nouislider": "^12.0.0",
```
Comment (amassarwi) [inline package.json:289]: 🎉

**Review by vctt94 — COMMENTED** [review]
Thanks for this PR, looking good, but I believe I am hitting an infinite loading which makes my computer really slow if I play a bit with the slider

**Comment by guilhermemntt** [comment]
Well, thanks for the feedback @vctt94, I'll take a look.

**Comment by guilhermemntt** [comment]
It seems it's possible to achieve better performance using  ``useCallback`` hook with lodash [debounce](https://lodash.com/docs/4.17.15#debounce). This way, we fetch from dcrwallet only after a 100 ms interval when sliding, avoiding infinite loadings, but this delay can take longer if needed.

**Comment by guilhermemntt** [comment]
Rebased.

**Comment by jholdstock** [comment]
I am getting errors when trying to run this with `yarn dev`. I can open the transaction history page, but when I try to open the "Sort" dropdown I get an electron console error:

```
Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function
(for composite components) but got: undefined. You likely forgot to export your component from the file it's
defined in, or you might have mixed up default and named imports.

Check the render method of `EyeFilterMenuWithSliderMenu`.
    in EyeFilterMenuWithSliderMenu (created by EyeFilterMenuWithSlider)
    in div (created by EyeFilterMenu)
    in div (created by EyeFilterMenu)
    in EyeFilterMenu (created by EyeFilterMenuWithSlider)
    in EyeFilterMenuWithSlider (created by Page)
    in div (created by Tooltip)
    in Tooltip (created by Page)
    ...
```

**Comment by guilhermemntt** [comment]
Thanks for testing this one @jholdstock. Could not reproduce this warning here. Are you sure you reinstalled the dependencies using ``yarn`` before running this branch?

**Review by jholdstock — APPROVED** [review]
That was it, it works after running `yarn`. Looks good and works well.

The performance is absolutely appalling when loading transactions - the app becomes unresponsive, CPU spins, and animations chug, but that is a known issue which is noted elsewhere, and not related to this PR.

**Comment by guilhermemntt** [comment]
Rebased.

File: package.json [hunk 2]
```diff
@@ -286,8 +286,7 @@
...
-    "pi-ui": "https://github.com/decred/pi-ui",
+    "pi-ui": "https://github.com/guilhermemntt/pi-ui#50b040d1ea0a5e83f6d2fd9d9f3a970282f23320",
```
Comment (alexlyp) [inline package.json:289]: This is a NACK for me.  Needs to be in a decred owned repo.  Or someone with longer standing.

**Review by alexlyp — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (guilhermemntt) [reply package.json:289]: Oh, @alexlyp, sorry. Guys at ``pi-ui`` will merge my PR there as soon this one here is approved, but I can talk to them. So I can use the master ``pi-ui`` repo and we can finally merge it if it's ok.

Comment (alexlyp) [reply package.json:289]: OK now that it's merged in pi-ui, please go ahead and remove this change and then should be good to go

Comment (guilhermemntt) [reply package.json]: Done! Thanks @alexlyp

File: package.json [hunk 3]
```diff
@@ -286,7 +286,8 @@
     "mv": "^2.1.1",
     "node-abi": "^2.15.0",
     "node-addon-loader": "decred/node-addon-loader#master",
-    "pi-ui": "https://github.com/guilhermemntt/pi-ui#50b040d1ea0a5e83f6d2fd9d9f3a970282f23320",
+    "nouislider": "^12.0.0",
```
Comment (amassarwi) [inline package.json]: This should go away

Comment (guilhermemntt) [reply package.json]: Done! Thanks!

**Review by alexlyp — APPROVED** [review]

**Pull request merged**